README.md

```diff
@@ -72,6 +74,14 @@ Before you do that you will need to connect to the RDS console query editor. Use
 
 By following all the steps above you'll have a table in the database to store profile entries into talent board.
 
+**7.** To deploy API docs to your AWS account, follow the steps given below:
```

Comment:
`To deploy the API docs to your AWS account, follow the steps given below:`

Reply:
Addressed this remark